.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*       @gaborbernat @asottile @obestwalter
```

Comment:
Removing other people from here so I don't spam people with pull requests review until this branch is ready to land.